cluster/common.go

```diff
@@ -142,13 +142,15 @@ func StoreKubernetesConfig(cluster CommonCluster, config []byte) error {
 	encodedConfig := utils.EncodeStringToBase64(string(config))
 
 	organizationID := cluster.GetOrganizationId()
+	clusterIdTag := fmt.Sprintf("clusterid: %d", cluster.GetID())
```

Comment:
Is there an extra space there? (based on `"app:grafana"`). Does that cause any problem?